<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>401</width>
     <height>155</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="lblDuration">
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lblArt">
      <property name="text">
       <string>Art</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnBreak">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Break</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lblEstimate">
      <property name="text">
       <string>Estemate</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QComboBox" name="cmbEstimateUnit">
      <item>
       <property name="text">
        <string>Minutes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hours</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="lblEstimateValue"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnStartStop">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="txtAddTime">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>20 minutes ago</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="chbAutoStart">
      <property name="text">
       <string>Auto-Start</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="txtProject">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Project Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QComboBox" name="cmbArt">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
